database/migrations/2021_06_30_122229_config_new_photos_notification.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class ConfigNewPhotosNotification extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		defined('BOOL') or define('BOOL', '0|1');
+
+		DB::table('configs')->insert([
+			[
+				'key' => 'new_photos_notification',
+				'value' => '0',
+				'confidentiality' => 0,
```

Comment:
I think this should be of level 2 (logged in) if I am not wrong.

Reply:
Yeh, I did try level 2 originally, however it doesn't seem to be allowing the config option to be accessible from a logged in (restricted) user, which is used to show/hide the notification menu item if the setting is enabled or not. All related to the Front-End. I could be missing something?

Reply:
Ummmm what about level 1 then ? :s
I don't remember exactly which level is which access, this should probably be clarified in the docs :)

Reply:
You'll know it better than me :) but when I've looked through it, I could only see logged in users receiving public configs (https://github.com/LycheeOrg/Lychee/blob/master/app/ModelFunctions/ConfigFunctions.php#L51) and admins receiving the admin configs. Although there is a scope for it: https://github.com/LycheeOrg/Lychee/blob/master/app/Models/Configs.php#L261